home work 24/script.js

```diff
@@ -0,0 +1,56 @@
+const productsApiUrl = "https://fakestoreapi.com/products";
+const productsContainer = document.querySelector(".products-container");
+const productDetails = document.querySelector(".product-details");
+
+function fetchProducts() {
+  fetch(productsApiUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      productsContainer.innerHTML = ""; 
+      const shuffledData = shuffle(data);
+      shuffledData.slice(0, 10).forEach((product) => {
```

Comment:
No need for a slice. You should request to `https://api.escuelajs.co/api/v1/products/?offset=0&limit=10` where the limit is a max count of products. Please be attentive to the task requirement